### ISSUE TITLE
module error in unet classification - problem and fix 

### ISSUE THREAD

**Issue by VolkerH** [CellProfiler/CellProfiler-plugins#70]
When pip-installing the requirements-windows.txt file, pip pulled keras version 2.2.2.

With this version, the classify-unet module will throw an error as the interface to `keras.layers.merge` seems to have changed recently.
 To fix it, one has to replace all calls of the kind `keras.layers.merge([], concat_axis=3. mode='concat')` to 

` keras.layers.merge.concatenate([e, b], axis=3)`.

**Comment by ErinWeisbart** [comment]
I am closing this issue because it refers to a plugin that we are not currently maintaining.
